src/utils.ts

```diff
@@ -655,3 +660,19 @@ export function fillErrorDetails(properties: IExecutionResultTelemetryProperties
         properties.exceptionId = e.id.toString();
     }
 }
+
+export function makeUnique<T>(elements: T[]): T[] {
+    return Array.from(new Set(elements));
+}
+
+export function adaptToSinglIntoToMulti<T, R>(thisObject: object, toSingle: (single: T) => R): (multi: T[]) => R[] {
```

Comment:
`Single`

Reply:
Can you add a comment for what this does? I don't really follow the intent. It's a helper for `map`?

Reply:
/**
 * adaptToSinglIntoToMulti transforms a function that takes a single parameter, to a version that takes an array and applies the function to each parameter.
 * Some usage examples:
 *   public readonly getBPsFromIDs = adaptToSinglIntoToMulti(this, this.getBPFromID);
 *   public readonly toStackFrames = asyncIStackTracePresentationRow(this, this.toStackFrame);
 *   public readonly toSourceTrees = asyncIStackTracePresentationRow(this, this.toSourceTree);
 *   public readonly toBPRecipiesStatus = asyncIStackTracePresentationRow(this, this.toBPRecipieStatus);
 */